components/script/dom/window.rs

```diff
@@ -1602,48 +1602,87 @@ impl WindowMethods<crate::DomTypeHolder> for Window {
         self.ScrollY()
     }
 
-    // https://drafts.csswg.org/cssom-view/#dom-window-scroll
+    /// <https://drafts.csswg.org/cssom-view/#dom-window-scroll>
+    ///
+    /// - Step 1: If invoked with one argument, follow these substeps:
+    /// - Step 1.1: Let options be the argument.
```

Comment:
I think it's weird that step 1.1 gets to be in the documentation of the method, while the rest are just comments inside. Probably should move this inside.